src/control/server/server.go

```diff
@@ -248,10 +250,22 @@ func Start(log *logging.LeveledLogger, cfg *Configuration) error {
 	go func() {
 		sig := <-sigChan
 		log.Debugf("Caught signal: %s", sig)
-		if err := drpcCleanup(cfg.SocketDir); err != nil {
-			log.Errorf("error during dRPC cleanup: %s", err)
```

Comment:
Why not clean up the dRPC cruft if we're shutting down?

Reply:
we do, just moved to when harness.Start() exits. we do the corresponding setup there so seems the appropriate place for it

Reply:
https://github.com/daos-stack/daos/pull/1878/files#diff-157575dd1e4f090676b955542b630dc4R371

Reply:
Oops, I got confused - forgot that Start itself runs until the server is ready to be shut down. Thanks for the clarification.